versions.json

```diff
@@ -7,5 +7,6 @@
 	"1.0.5": "0.15.0",
 	"1.0.6": "0.15.0",
 	"1.0.7": "0.15.0",
-	"1.1.0": "0.15.0"
+	"1.1.0": "0.15.0",
+	"1.1.1": "0.15.0"
```

Comment:
Can you also run npm install such that the lock file updates? The version is not updated there.